### PR TITLE
SQL procedures updated

### PR DESCRIPTION
**Pull request by atharv14**
SQL procedures are executing successfully with springboot

### CODE REVIEW

File: srs-api/src/main/resources/sql/schema_procs.sql [hunk 1]
```diff
@@ -1,3 +1,405 @@
...
+
+PROCEDURE LIST_CLASS(CID IN CLASSES.CLASSID%TYPE, CURSOR_OUTPUT OUT REF_CURSOR) IS
+    CLASS_COUNT NUMBER;
+BEGIN
+SELECT COUNT(*) INTO CLASS_COUNT FROM CLASSES c WHERE c.CLASSID = CID;
+IF CLASS_COUNT = 0 THEN
+        raise_application_error(-20001, 'The classid: ' || CID || ' is invalid.');
+END IF;
+OPEN CURSOR_OUTPUT FOR
+SELECT s."B#", s.FIRST_NAME, s.LAST_NAME
```
Comment (r15hank) [inline srs-api/src/main/resources/sql/schema_procs.sql]: This should have been inner joined

File: srs-api/src/main/resources/sql/schema_triggers.sql [hunk 2]
```diff
@@ -14,3 +14,83 @@ BEGIN
...
+     (SELECT USER FROM dual),
+     CURRENT_TIMESTAMP,
+     'STUDENTS',
+     'DELETE',
+     :OLD.B#);
+END;
+/
+show errors
+
+CREATE OR REPLACE
```
Comment (r15hank) [inline srs-api/src/main/resources/sql/schema_triggers.sql]: there should be seperate triggers for logging and enrolling/de-enrolling

File: srs-api/src/main/resources/sql/schema_triggers.sql [hunk 3]
```diff
@@ -14,3 +14,83 @@ BEGIN
...
+VALUES
+    (logseq.NEXTVAL,
+     (SELECT USER FROM dual),
+     CURRENT_TIMESTAMP,
+     'STUDENTS',
+     'DELETE',
+     :OLD.B#);
+END;
+/
+show errors
```
Comment (r15hank) [inline srs-api/src/main/resources/sql/schema_triggers.sql]: / show errors shouldn't be mentioned here

File: srs-api/src/main/resources/sql/schema_triggers.sql [hunk 4]
```diff
@@ -14,3 +14,83 @@ BEGIN
...
+                'DELETE',
+                TO_CHAR (:OLD.G_B#) || ',' || TO_CHAR (:OLD.CLASSID));
+UPDATE
+    CLASSES c
+SET
+    c.CLASS_SIZE = c.CLASS_SIZE  - 1
+WHERE
+        c.CLASSID = :OLD.CLASSID;
+END IF;
+END;
```
Comment (r15hank) [inline srs-api/src/main/resources/sql/schema_triggers.sql]: all procedural block statements need to end with ";^"

**Review by r15hank — CHANGES_REQUESTED** [review]
Needs few changes

File: srs-api/src/main/resources/sql/schema_triggers.sql (hunk 4 above)
Comment (atharv14) [reply srs-api/src/main/resources/sql/schema_triggers.sql]: when executed adding ";^" is interpreted as "; ;" giving sql syntax error

**Review by r15hank — APPROVED** [review]

**Pull request merged**